<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadedImagesForm</class>
 <widget class="QWidget" name="LoadedImagesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>205</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>120</width>
    <height>205</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CLoadedImagesWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="imageFile">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;File Name&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Unchanged</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BC1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BC2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BC3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BC6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BC7</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>151</y>
     <width>120</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Export As</string>
   </property>
  </widget>
  <zorder>imageFile</zorder>
  <zorder>groupBox</zorder>
  <zorder>comboBox</zorder>
  <zorder>openGLWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CLoadedImagesWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>Src/Widgets/LSSELoadedImagesWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
